frontend/src/Frontend.hs

```diff
@@ -53,7 +54,7 @@ frontend = Frontend
   , _frontend_body = prerender_ loaderMarkup $ do
     (fileOpened, triggerOpen) <- openFileDialog
     let store = browserStorage
-        crypto = Crypto mkSignature (const genKeyPair)
+        crypto = Crypto mkSignature (const genKeyPair) deriveKeyPairFromPrivateKey
```

Comment:
Don't forget the same change in Desktop bipWallet! You are then gonna have to deal with XPrv there so it'll be harder. 

It may actually be worth just using the Pact / Cryptonite stuff there given we had to change the crypto api anyway...

https://github.com/kadena-io/chainweaver/pull/264/files#diff-152ee524a8f6733eb8014b53f0ebbf73R150